articles/openshift/oshift-how-restrict-access-to-openshift-routes-by-ip.md

```diff
@@ -0,0 +1,57 @@
+---
+title: UKCloud Knowledge Centre article about whitelisting public IP's on your openshift routes | UKCloud Ltd
+description: Restrict access to openshift routes by public IP[s]
```

Comment:
Change openshift to OpenShift

(Make this change throughout the article. It's important that we get the names of third party products correct, including the correct capitalisation as quite often they are trademarks)